.gitignore

```diff
@@ -0,0 +1,2 @@
+/quakespasm/Windows/VisualStudio/.vs/
+/quakespasm/Windows/VisualStudio/Build-quakespasm-sdl2/
```

Comment:
When I compiled and debugged on this branch, visual studio created the files "history.txt" and "quakespasm-sdl2.vcxproj.user" in the VisualStudio directory which were then marked as untracked files by git, so I'd suggest these too:

```
/quakespasm/Windows/VisualStudio/history.txt
/quakespasm/Windows/VisualStudio/*.user
```

(vkQuake also has gitignores for these)